particle_num: 1000

alpha_1: 0.001   # uncertainty weight
alpha_2: 0.001
alpha_3: 10.0
alpha_4: 0.01
init_pos_conv: 2.0
init_orient_conv: 0.1